UIMenuItem.cs

```diff
@@ -108,26 +109,26 @@ public virtual void Draw()
 
             if (Hovered && !Selected)
             {
-                _rectangle.Color = Color.FromArgb(20, 255, 255, 255);
+                //_rectangle.Color = Color.FromArgb(20, 255, 255, 255); // Why setting color every time? (I set it in ctor)
```

Comment:
When you hover with your mouse over a menu Item, it would darken the item, it still does, just not repeated any more.